modules/UI/util/UIUtil.js

```diff
@@ -103,13 +103,17 @@ import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
             'top-right': 'sw'
         };
 
-        element.setAttribute("data-i18n", "[content]" + key);
+        $(element).each(function () {
+            this.setAttribute("data-i18n", "[content]" + key);
+        });
         APP.translation.translateElement($(element));
 
         AJS.$(element).tooltip({
             gravity: positions[position],
             title: this._getTooltipText.bind(this, element),
-            html: true
+            html: true,
+            live: true,
+            hoverable: false
```

Comment:
Isn't the default value of `hoverable` `false` already? If it is, could you please not specify it. Or is it really necessary?